colossalai/kernel/cuda_native/csrc/transpose_pad_fusion_kernel.cu

```diff
@@ -0,0 +1,93 @@
+// For eliminating redundant computation.
+// The core idea and code of kernel is from TurboTransformer.
```

Comment:
Be careful. If you use the code you should use the copyright of Tencent.
You'd better polish the code and reimplement it by yourself.
You should also add the necessary unitest for your CUDA kernels.

Reply:
I'll add the copyright of Tencent and convert my test into the unitest format.
Since I am working on the inference engine, I plan to apply the "transpose_pad" method as a demo policy as well as the first policy for dealing with redundant computation.